wallet/src/ui/app/hooks/useFormatDate.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// TODO - handle multiple date formats
+// Wed Aug 05
+export default function useFormatDate(date: Date) {
+    return new Date(date).toLocaleDateString('en-us', {
```

Comment:
If you pass `undefined` as the first argument it should use the user's preferred locale, which might be nice.

Also, you don't need to date wrap date, as it's already a date. Should just be able to do `date.toLocaleDateString()`

Reply:
Still working on this. Thanks for the feedback

Reply:
This is done, review